registry.k8s.io/images/k8s-staging-kueue/images.yaml

```diff
@@ -1,5 +1,6 @@
 - name: charts/kueue
   dmap:
+    "sha256:e5bbf2a32c188f2d8760d4e7e793b8af6e40127c8d53aea1a247a81e128bb471": ["0.11.4"]
```

Comment:
> helm template oci://us-central1-docker.pkg.dev/k8s-staging-images/kueue/charts/kueue --version 0.11.4 | grep registry.k8s.io 
        image: "registry.k8s.io/kueue/kueue:0.11.4"

Reply:
Good point.

Reply:
Currently, @mimowo is working on fixing releaseing script